memory/src/main/scala/filodb.memory/format/vectors/Histogram.scala

```diff
@@ -324,6 +327,91 @@ final case class MaxHistogram(innerHist: MutableHistogram, max: Double) extends
   }
 }
 
+final case class DDSHistogram(relativeAccuracy: Double) extends HistogramWithBuckets {
+  def serialize(intoBuf: Option[MutableDirectBuffer] = None): MutableDirectBuffer = ???
+
+  val logGamma: Double = Math.log((1 + relativeAccuracy) / (1 - relativeAccuracy))
+  val maxIndexedValue = maxIndexableValue
```

Comment:
This seems to be a self-reference.  I see later there is a method named the same, but for clarify you should rename the method, or since this is just initialized once, you could probably move the init code to up here.